beta-0.44

lochner-lw opened the pull request.

Commit:
Merge branch 'alfe/Sterling/prod/main' into alfe/Sterling/dev/new_defaults-042825

Commit:
- Updated default AI model from  to .
  • When a new chat is created.
  • Whenever an existing chat lacks an explicit model.
  • When the server falls back to a model during chat-message handling.

Commit:
modified:   executable/server_webserver.js

lochner-lw committed:
remove comments

Commit:
DEFAULT_AIMODEL

Commit:
..

Commit:
• New chats now have “Push after commit” enabled by default.
   ‑ Updated route  in  –  is set to  instead of .

Commit:
Revert "• New chats now have “Push after commit” enabled by default."

This reverts commit 5505d5cad72667be7bf70fb94ccaa7a06828f888.

Commit:
default

Commit:
fix

Commit:
[DEBUG]

lochner-lw committed:
Merge pull request #9 from alfe-ai/alfe/Sterling/dev/new_defaults-042825

new defaults

Commit:
Assistant (4/28/2025, 10:36:54 PM)
A. Commit Summary
1. Added SSH-to-HTTPS conversion so that GitHub links inside the chat page always use a proper https:// URL rather than the SSH form.
   • Introduced helper `convertGitUrlToHttps()` in `server_webserver.js`.
   • Applied the helper when preparing `githubURL` for the chat view, ensuring links such as “GitHub” and branch links point to e.g. `https://github.com/alfe-ai/alfe` instead of `git@github.com:alfe-ai/alfe.git`.

Commit:
fix attempt

Commit:
.

Commit:
.

Commit:
nononsense

Commit:
• Revised server_webserver.js to incorporate Git URL conversion logic and other enhancements.

nicholelochner commented on executable/server_webserver.js:
```diff
@@ -922,10 +923,10 @@ app.post(
                     // Set git commit username and email
-                    execSync('git config user.name "whimsy"', {
+                    execSync('git config user.name "YOURNAME"', {
                         cwd: gitRepoLocalPath,
                     });
-                    execSync('git config user.email "whimsy@sylph.box"', {
+                    execSync('git config user.email "YOURNAME@YOURDOMAIN.tld"', {
                         cwd: gitRepoLocalPath,
```
Moved to https://github.com/alfe-ai/alfe/issues/12

nicholelochner commented on executable/server_webserver.js:
```diff
@@ -922,10 +923,10 @@ app.post(
                     // Set git commit username and email
-                    execSync('git config user.name "whimsy"', {
+                    execSync('git config user.name "YOURNAME"', {
                         cwd: gitRepoLocalPath,
                     });
-                    execSync('git config user.email "whimsy@sylph.box"', {
+                    execSync('git config user.email "YOURNAME@YOURDOMAIN.tld"', {
                         cwd: gitRepoLocalPath,
```
Moved to https://github.com/alfe-ai/alfe/issues/12

lochner-lw committed:
Merge pull request #11 from alfe-ai/alfe/Sterling/dev/new_defaults-042825-github_urls

alfe/Sterling/dev/new_defaults-042825-github_urls

Commit:
Fixed the hardcoded git commit configuration by externalizing the commit username and email into environment variables. Users can now set GIT_COMMIT_USER_NAME and GIT_COMMIT_USER_EMAIL in their environment or .env file to override these defaults.

Commit:
Revert "Fixed the hardcoded git commit configuration by externalizing the commit username and email into environment variables. Users can now set GIT_COMMIT_USER_NAME and GIT_COMMIT_USER_EMAIL in their environment or .env file to override these defaults."

This reverts commit 8f3977ff4f9ed3d8822bb4000071585035cb1101.

Commit:
• Fixed the hardcoded git commit configuration by externalizing the commit username and email into environment variables.
• Users can now set GIT_COMMIT_USER_NAME and GIT_COMMIT_USER_EMAIL in their environment or .env file to override these defaults.

lochner-lw committed:
Merge pull request #14 from alfe-ai/12-hardcoded-placeholder-values-for-the-git-commit-username-may-lead-to-misidentification-in-commits-consider-externalizing-this-configuration-for-flexibility

12 hardcoded placeholder values for the git commit username may lead to misidentification in commits consider externalizing this configuration for flexibility
